src/app/components/FrostedGlassPromo/__snapshots__/index.test.jsx.snap

```diff
@@ -119,63 +98,112 @@ exports[`Frosted Glass Promo when given props directly 1`] = `
   }
 }
 
+.emotion-15:focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.emotion-17 {
+  display: none;
+}
+
+@supports (filter: blur(15px)) {
+  .emotion-17 {
+    display: block;
+    z-index: 1;
+    position: absolute;
+    bottom: 0;
+    top: -15px;
+    left: 0;
+    right: 0;
+    background: #202224 url('https://ichef.bbci.co.uk/news/400/cpsdevpb/DDFC/test/_63482865_orange2.jpg');
+    background-repeat: no-repeat;
+    -webkit-background-size: cover;
+    background-size: cover;
+    -webkit-background-position: bottom;
+    background-position: bottom;
+    -webkit-transform: scaleX(1.15) scaleY(-1.15);
+    -moz-transform: scaleX(1.15) scaleY(-1.15);
+    -ms-transform: scaleX(1.15) scaleY(-1.15);
+    transform: scaleX(1.15) scaleY(-1.15);
+    -webkit-filter: blur(15px);
+    filter: blur(15px);
+  }
+}
+
 <div>
-  <a
+  <div
     class="emotion-0 emotion-1"
-    href="/mundo/internacional-23038380"
   >
-    <div
+    <a
```

Comment:
"Error: Self-closing syntax (/>) used on a non-void HTML element. Ignoring the slash and treating as a start tag."

Self closing a tags aren't valid HTML

Reply:
👍  will update this